payjoin-io/README.md

```diff
@@ -0,0 +1,6 @@
+# payjoin-io
+
+This repository contains a collection of I/O utilities for working
+with `rust-payjoin` crate. It is a work in progress and will be
+updated as new tools are developed.
```

Comment:
This doesn't open a new repository, just a library. We may post this to crates.io so this should be catered to an audience there. I also get sad any time I come to a website that says WIP/coming soon. Just tell me what it does! All software is WIP. This isn't a draft PR, it's published software ;)

Try something like

> A collection of I/O utilities for working with the [`payjoin`](https://crates.io/crates/payjoin) crate.
> The `payjoin` crate only deals with the low-level protocol. `payjoin-io` provides sane defaults to make implementing payjoin easy.